composer.lock

```diff
@@ -4,32 +4,32 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
```

Comment:
Do not add it to this PR

Reply:
Although I have removed this file, it will break the next test, so please keep that in mind: 

https://github.com/elementor/elementor/actions/runs/3805233558/jobs/6473031108